RecoHGCal/TICL/plugins/PatternRecognitionbyCA.cc

```diff
@@ -446,7 +436,7 @@ void PatternRecognitionbyCA<TILES>::energyRegressionAndID(const std::vector<reco
   }
 
   // run the inference (7)
-  tensorflow::run(eidSession_, inputList, outputNames, &outputs);
+  tensorflow::run(const_cast<tensorflow::Session *>(eidSession), inputList, outputNames, &outputs);
```

Comment:
`eidSession` is already a `const tensorflow::Session *`: why do you need the `const_cast`?

Reply:
this is removing the const (but tensorflow::run is known to be thread-safe), to fit the the tensorflow interface.
related to https://github.com/cms-sw/cmssw/issues/35541